docs/reference/troubleshooting/troubleshooting-shards-capacity.asciidoc

```diff
@@ -0,0 +1,104 @@
+[[troubleshoot-shards-capacity-issues]]
+== Troubleshoot shards capacity health issues
+
+Elastic limits the maximum number of shards to be held per node using the
+<<cluster-max-shards-per-node, `cluster.max_shards_per_node`>> and
+<<cluster-max-shards-per-node-frozen, `cluster.max_shards_per_node.frozen`>> settings.
+The current shards capacity of the cluster is available in the
+<<health-api-response-details-shards-capacity, health API shards capacity section>>.
+
+[discrete]
+=== Cluster is close to reaching the configured maximum number of shards for data nodes.
+
+The <<cluster-max-shards-per-node,`cluster.max_shards_per_node`>> cluster
+setting limits the maximum number of open shards for a cluster, only counting over data nodes
+that do not belong to the frozen tier.
+
+This symptom indicates that an action should be taken, otherwise, either the creation of new
+indices or the upgrading of the cluster could be blocked.
```

Comment:
```suggestion
This symptom indicates that action should be taken, otherwise, either the creation of new
indices or upgrading the cluster could be blocked.
```